### DISCUSSION TITLE
告别信：离开NextTrace的决定

### DISCUSSION THREAD

**Discussion by sjlleo** [nxtrace/NTrace-core#214] [General]
大家好，

经过一段时间的考虑，我决定退出NextTrace开源项目。近来我发现，在NextTrace的未来发展和核心理念上，我和其他成员有着显著的不同看法。同时，我的时间和精力也越来越无法支撑我继续全身心地投入这个项目。

在NextTrace项目中的时间对我来说是宝贵的，我有幸与一些非常才华横溢的人合作。但如今，由于我们在项目方向和实施细节上的分歧，我认为最好的选择是让路给那些更符合项目当前需求的人。

我将在接下来的几天里完成任务交接，并确保所有相关文档和代码能顺利传递给现任主要项目Owner @tsosunchia 。

虽然我即将离开，但我会继续关注NextTrace，希望它能取得更大的成功。

感谢大家的支持和理解。

最诚挚的问候，
Leo Shen

**Comment by tsosunchia** [comment]
？？？

**Comment by missuo** [comment]
It was a tough decision and I hope NextTrace gets better.

**Comment by yhyu13** [comment]
https://zhuanlan.zhihu.com/p/655297665
